prisma/schema.prisma

```diff
@@ -11,8 +11,18 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Post {
-  id    String  @id @default(auto()) @map("_id") @db.ObjectId
-  title String  @unique
-  body  String
+// model Post {
+//   id    String  @id @default(auto()) @map("_id") @db.ObjectId
+//   title String  @unique
+//   body  String
+// }
+
+model Cryptocurrency {
+  id   Int   @id @map("_id")
+  name String 
+}
+
+model MapData {
+  id    String   @id @map("_id") 
```

Comment:
**nit:** What is map data? Shoudnt it have more specified structure?

Reply:
For simplier implementation i've created only one collection containing mapped data of crypto's and fiat, hence these had so much different schemes i've gone for simple JSON typing, and intended to distinguish them by id and associated methods to these id's. 
Nevertheless as you noticed, it would prove to be more readable to create 2 distuinguishable collections with clear schemas, i'll do this then.